core/utils/load_batch.py

```diff
@@ -205,40 +180,37 @@ def read_batch_sample_file(batch_file):
 def project_validation(sample_batch_data, app_name):
     """_summary_
 
-    Parameters
-    ----------
-    sample_batch_data
-        Json object with the samples batch data
-    app_name
-        Aplication name (wetlab, drylab)
+    Args:
+        sample_batch_data (pandas dataframe): data from user input file
+        app_name (string): name of the application
 
-    Returns
-    -------
-    error_cause
-        If date was not validated, returns the error.
+    Returns:
+        dict: Contains the result and optional the error message or the project
+        name
     """
+    validation_result = {"result": "NOK"}
```

Comment:
same as before, let's try to avoid the "OK/NOK"
If you want to create a structure with the validation result, include the structure in the docstring. If that's the case, you can include in the structure also de project_name